### PR TITLE
Further rework models and added a few new ones

### LINKED ISSUE #46: Zebra-Witch
We allredy made zebra-witch on 1.7.10, but it was not very good. We appdate our server to 1.12.2 and decided to propose you replace vanilla witch to zebras.
image

Administrator Minecraft for Everypony.

### PR DESCRIPTION
**Pull request by Sollace**
This PR includes the following changes:

- separate ModelAlicorn into ModelUnicorn, ModelPegasus, and ModelAlicorn
- Added Zebra models

  They have a longer neck and a brush cut on their manes. Texturing is identical to texturing wings on a pegasus.
- Finished Witches and implemented a proper drinking animation with the zebra model. Closes #46
https://imgur.com/NCRKKnC
- Guardians are now seaponies; look at them go!

 Leg positions has been changed since, though, so they look a bit like they're dancing on land (shooby doo), and will reach forwards depending on whether they're standing on a block under water.

### CODE REVIEW

File: src/main/resources/assets/minelittlepony/lang/fr_fr.lang [hunk 1]
```diff
@@ -10,10 +10,12 @@ minelp.options.hd=Activer MineLP serveur de skin
 minelp.options.sizes=Autoriser tous les diff�rentes tailles de poney
 minelp.options.ponyarmor=Utiliser armure compatible de MineLP
 minelp.options.snuzzles=Afficher museau sur les poneys
-minelp.options.showscale=Utiliser �chelle fid�le � MLP
```
Comment (Sollace) [inline src/main/resources/assets/minelittlepony/lang/fr_fr.lang]: Minecraft doesn't render these characters, so we'd might as well switch to the non-tilda variants.

Comment (killjoy1221) [reply src/main/resources/assets/minelittlepony/lang/fr_fr.lang]: Fix your encoding. Use UTF-8.

**Comment by killjoy1221** [comment]
I was thinking of having the sea ponies be a player model. Think you can do that? Same with Zebra. That sounds neat.

**Comment by killjoy1221** [comment]
Also, how do the sea ponies look underwater? Do they actually swim?

**Comment by Sollace** [comment]
@killjoy1221 I'm not really sure how the seaponies would work as a player model... They don't really have a leg to stand on there.

The seaponies do swim, the tail will just continue to flap at a constant rate. They also adjust their arms to point forwards when underwater.

And the Zebras are already player models. ;3

File: src/main/resources/assets/minelittlepony/lang/fr_fr.lang (hunk 1 above)
Comment (Sollace) [reply src/main/resources/assets/minelittlepony/lang/fr_fr.lang]: @killjoy1221 I thought I was using uft8? The unchanged file still didn't render when reading the text in-game. I'll double-check.

**Comment by TheWeatherPony** [comment]
I'll try to get in there and change some of that (letting players use it).

**Comment by PolyacovYury** [comment]
While you're at it, maybe fix (this line)[https://github.com/MineLittlePony/MineLittlePony/blob/58cac7e7f1da3a79a91db0ba8d1a421b516520ca/src/main/java/com/minelittlepony/model/ponies/ModelVillagerPony.java#L78]? `.tex(56, 29)` really should say `.tex(56, 19)`, otherwise villager bags are totally broken and need to be redrawn..

**Comment by Sollace** [comment]
@polyacovyury My last PR included changes to all the models, so I'll go check that. I had to rearrange the villager's saddlebags quite a bit to fix texture and mirroring issues.

One side I _was_ unsure about was the correct orientation of the flaps on the outside.

**Comment by Sollace** [comment]
@TheWeatherPony My point about seaponies as player models (I did consider it) was that, IMO, it wouldn't really make any sense. They look kind of stupid on land.

**Comment by PolyacovYury** [comment]
Well, ponifying guardians also looks pretty unnecessary. What will go next? Pony-squids? Pony-fish? :D

**Comment by Sollace** [comment]
@polyacovyry Pony Endermen? Pony Dragons?

How about a pony block?

**Comment by PolyacovYury** [comment]
Well, dk about pony dragons or blocks, but Pony-Endermen seem reasonable.
About player-seaponies - how about triggering transformation from hypogriff to a seapony when swimming animation is activated, and then back?

**Comment by Sollace** [comment]
@PolyacovYury That would be a possibility. I'd floated the idea of letting ponies switch when in the water, but adding hippogryphs makes more sense.

**Comment by Sollace** [comment]
@PolyacovYury Sounds nice. I'd considered an idea of implementing "accessories." Essentially you' have a trigger pixel that lets you specify whether your pony is wearing any one of the villager's equipment or the Witche's hat

The issue, though, is that you wouldn't be able to skin these items. I don't know if we have space on the regular skin.

If you want, please go and create an issue for each of these and we can discuss them further there.(this chat is really only about the current PR, for which @killjoy1221 has been MIA at least two weeks)

**Comment by Sollace** [comment]
@killjoy1221 Rebased onto the new master.

**Pull request merged**